sendgrid.go

```diff
@@ -42,6 +42,20 @@ func GetRequest(key string, endpoint string, host string) rest.Request {
 	return request
 }
 
+// GetRequests returns a default request object with multiple ApiKeys.
+func GetRequests(keys []string, endpoint string, host string) []rest.Request {
+	if host == "" {
+		host = "https://api.sendgrid.com"
```

Comment:
I see this is now used a couple times in file.  Would you add it to the const above?

Reply:
sorry, what do you mean?

Reply:
@alimasyhur,

I believe @robertacosta is asking for you to define "https://api.sendgrid.com" as a constant.